octobot_trading/exchanges/connectors/ccxt_exchange.py

```diff
@@ -142,9 +142,11 @@ def _create_client(self):
                 elif not self.exchange_manager.is_simulated:
                     self.logger.warning(f"No exchange API key set for {self.exchange_manager.exchange_name}. "
                                         f"Enter your account details to enable real trading on this exchange.")
-                self.client = self.exchange_type(self._get_client_config(key, secret, password))
                 if self._should_authenticate():
+                    self.client = self.exchange_type(self._get_client_config(key, secret, password))
                     self.client.checkRequiredCredentials()
+                else:
+                    self.client = self.exchange_type(self._get_client_config())
```

Comment:
Why do we need to do that now?

Reply:
To avoid giving api keys to exchanges created for simulated trading (which create errors when keys are invalid or computer time is out of synch)

Reply:
(its not a new problem, I just only realized it now)